src/main/java/UseCase/createPlaylist.java

```diff
@@ -5,27 +5,56 @@
 import InputData.newPlaylistInputData;
 import Database.playlistAccessInterface;
 import Database.playlistLibrary;
-public class createPlaylist implements newPlaylistInputBoundary {
+import OutputBoundary.newPlaylistOutputBoundary;
+import OutputData.newPlaylistOutputData;
+
+//import OutputBoundary.newPlaylistOutputBoundary;
```

Comment:
if you could remove commented out imports / functions, that would be great as it would make reading your code a lot easier.